src/PointwiseFunctions/GeneralRelativity/TagsCollections.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/*!
+ * \file
+ * \brief Defines lists of `gr::Tags`.
+ *
+ * With this separate file we avoid having to include `Tags.hpp` where it's
+ * unnecessary and we avoid including prefix tags in `TagsDeclarations.hpp`.
+ */
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+
+namespace gr {
+namespace Tags {
+
+/// All tags for spacetime quantities in the 3+1 decomposition
+template <typename DataType, size_t Dim, typename Frame = Frame::Inertial>
+using all_spacetime_three_plus_one = tmpl::list<
```

Comment:
I don't like this name because it's not true and super dependent on what you think are the core 3+1 quantities. And then the other question is: why not include spatial christoffel symbols? Those are part of "all".